DataLakeModels/Models/YouTube/Studio/Video.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+
+namespace DataLakeModels.Models.YouTube.Studio {
+
+    public class Video : IValidityRange, IEquatable<Video> {
+
+        public Video(
+            DateTime validityStart,
+            DateTime validityEnd,
+            DateTime dateMeasure,
+            string channelId,
+            string videoId,
+            string metric,
+            double value
+        )
+        {
+            ValidityStart = validityStart;
+            ValidityEnd = validityEnd;
+            DateMeasure = dateMeasure;
+            ChannelId = channelId;
+            VideoId = videoId;
+            Metric = metric;
+            Value = value;
+        }
+        public Video(){}
+
+        public DateTime ValidityStart { get; set; }
+        public DateTime ValidityEnd { get; set; }
+        /* See "DateOnly" type introduced in .NET 6 */
```

Comment:
You can add a tag to the DateTime so that in PostgreSQL it will know it is a Date
```c#
[Column(TypeName = "date")]
```